src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs

```diff
@@ -48,9 +29,7 @@ public string FullScript
             {
                 if (!string.IsNullOrEmpty(this.telemetryConfiguration.InstrumentationKey))
                 {
-                    return ScriptPrefix + @"{
-  instrumentationKey: '" + this.telemetryConfiguration.InstrumentationKey + @"'
-}" + ScriptPostfix;
+                    return string.Format(Resources.JavaScriptSnippet, this.telemetryConfiguration.InstrumentationKey);
```

Comment:
It is cached in the resource manager I believe (it's been a while since I read that code in Reflector...).